### PR TITLE
GRAS LC 21+22 relocation

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Changing location codes of two sites at GRAS
 - location 21 -> 23
 - location 22 -> 24

### CODE REVIEW

**Review by CBurton90 — APPROVED** [review]

**Comment by CBurton90** [comment]
Posed a question in delta regarding dealing with location code changes at building arrays but this looks fine to merge.

**Review by salichon — COMMENTED** [review]
if location code change lat/lon sites shoudl reflect it accordingly ?

**Review by mnaguit — COMMENTED** [review]
Thanks @Thomas-Benson , I did the first review. But I'll wait for confirmation on other station metadata before I approve (recorder azimuth, site elevation and coordinates), hope that is fine.

**Comment by CBurton90** [comment]
@salichon @mnaguit @Thomas-Benson Locations 23 and 24 run with PTP timing supplied by location 20 (which is a PTP master clock and connected to a GNSS antenna). The GNSS coordinates (based off the antenna) for location 20 is the same as previously, and locations 23 and 24 do not have a GNSS determined coordinates (as they are running of PTP with no GNSS antenna connected to them).

If we want new coordinates for locations 23 and 24 we will have to guestimate it off the sketch Paul left in the ticket. Happy to do this, let me know.

**Comment by salichon** [comment]
@CBurton90 yeap that be great i reckon for clarity. 
Thanks !

**Review by mnaguit — DISMISSED** [review]

**Comment by Thomas-Benson** [comment]
Updated site locations based on the positions of the sensors relative to the building layout + topographic map
GRAS 21: -38.664355 178.025033
GRAS 22: -38.664406 178.025120
GRAS 23 & 24: -38.664453 178.025186

**Review by CBurton90 — DISMISSED** [review]

File: network/sites.csv
Comment (mnaguit) [inline network/sites.csv:962]: latitude for 21 is `178.025033`?

Comment (Thomas-Benson) [reply network/sites.csv]: Fixed now sorry, looks like I accidentally deleted the comma between longitude and elevation for that extra 26 to get there.

Comment (mnaguit) [reply network/sites.csv]: no worries, thanks @Thomas-Benson 👍

**Review by CBurton90 — APPROVED** [review]

**Review by mnaguit — APPROVED** [review]
Can you please remind me: I think for GRAS, we already discussed the HN-12Z streams vs recorder azimuth (aligned to north)? Approving this now, nonetheless.

**Comment by Thomas-Benson** [comment]
@mnaguit I'm not sure if the stream names have been discussed for this station recently, but it looks like all three of the the recorders here have always had an orientation of 0 and HN-12Z stream codes. It could be worth changing the stream codes to HN-ENZ since they've always been orientated to north, but we would need to make that decision for all of the past streams at this site too.

**Comment by CBurton90** [comment]
@Thomas-Benson @mnaguit I think we've all agreed we should keep building/urban based strong motion channels as axial naming (HNZ|1|2) even if the recorder is aligned north. 

https://github.com/GeoNet/delta/blob/main/docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md

 Perhaps I should add a line to mention building arrays for clarity?

**Comment by Thomas-Benson** [comment]
Thanks @CBurton90. That was my gut instinct too, but I know we've changed our stance on how we treat these changes in orientation several times and I could't find our most recent descision.

Mentioning building arrays is a good idea, although this site was not originally a proper building array where the sensors would be aligned with the structure, which I think is where part of the confusion comes from.

**Comment by CBurton90** [comment]
@Thomas-Benson All true! Yes we seem to constantly get tripped up a bit on the axial vs non-axial naming for strong motion sites and building arrays. I'll add a line to that doc to incorporate building arrays just so we make it clear. Will tag you both in the PR. Thanks.

**Comment by mnaguit** [comment]
Hi @CBurton90 @Thomas-Benson , did I get it right that for this case, the building is aligned to true north?

**Comment by Thomas-Benson** [comment]
@mnaguit No the building is not north aligned, but the sensor are not aligned with the structure here
image

**Comment by CBurton90** [comment]
@mnaguit @Thomas-Benson Looking at the photos for GRAS LC 24 it seems there is still some confusion from techs for orienting recorders in buildings. Looks like it has been forced at an angle to align with north rather than using the existing building/structure orientation.

20240621_023942866_iOS
20240620_045822990_iOS

**Comment by mnaguit** [comment]
@CBurton90 @Thomas-Benson 
Let's document this discussion in the main ticket, and identify the next steps. Thanks.

**Comment by salichon** [comment]
Are we missing anything here ? :)

**Comment by CBurton90** [comment]
> Are we missing anything here ? :)

@salichon I think the PR accurately represents the state of the site in its present form. Thomas has provided updated coordinates for 23/24 based on google maps/earth and Paul's sketches.

 I'd say we merge this as it will be a while before there is a return visit to align the recorders to the building structure.

**Comment by Thomas-Benson** [comment]
Sounds goo, we'll work out what to do with the orientations next time I guess

**Pull request merged**
